.gitignore

```diff
@@ -1,2 +1,3 @@
+.DS_Store
```

Comment:
Since these files come from the system rather than this project, I think this makes most sense in an `excludesfile` in the global `gitconfig`. (See https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer.) However it’s not a problem duplicating this here.

Reply:
Ah, I didn’t know about that global Git ignore list! Thanks.
Anyway, including `.DS_Store` will make life easier for everyone who is not aware of this (like me, until 10 minutes ago). We can also assume that this repo will mostly be used on Macs, so… :)